docs/examples.rst

```diff
@@ -48,12 +48,24 @@ can be used, called ``MoabCluster``:
                          resource_spec='pmem=96G',
                          job_extra=['-d /home/First.Last', '-M none'],
                          local_directory=os.getenv('TMPDIR', '/tmp'))
-                        
+
 SGE Deployments
 ---------------
 
 Examples welcome `here <https://github.com/dask/dask-jobqueue/issues/40>`_
 
+On systems which use SGE as the scheduler, ```SGECluster`` can be used:
+
+.. code-block:: python
+
+    from dask_jobqueue import SGECluster
+
+
+    cluster = SGECluster(queue='default.q',
+                         walltime="1500000",
+                         processes=10,
+                         memory='20GB')
```

Comment:
I'm curious, you don't specify anything else when launching SGECluster?  This section is intended to be filled with real-world examples, so we encourage you not to strip things out for simplicity's sake.

Reply:
Alright.  Thanks for sharing! 

In case you haven't already read through it, http://dask.pydata.org/en/latest/setup/hpc.html might be helpful.  I've found that it's common for people to want to specify network interfaces and local directories in particular.